jerry-libc/CMakeLists.txt

```diff
@@ -77,10 +77,10 @@ set(COMPILE_FLAGS_LIBC "${COMPILE_FLAGS_JERRY} ${C_FLAGS_JERRY}")
 
  # Platform-specific
   # Linux
-   file(GLOB SOURCE_LIBC_LINUX target/linux/*.c target/linux/*.S)
+   file(GLOB SOURCE_LIBC_LINUX target/posix/*.c target/posix/*.S)
 
   # Darwin
-   file(GLOB SOURCE_LIBC_DARWIN target/darwin/*.c target/darwin/*.S)
+   file(GLOB SOURCE_LIBC_DARWIN target/posix/*.c target/posix/*.S)
```

Comment:
No, because we have non-posix targets as well, MCU_STM32F3 and MCU_STM32F4. They have their corresponding `SOURCE_LIBC_MCU_STM32F3` and `SOURCE_LIBC_MCU_STM32F4` variables and use the implementation in `target/mcu-stubs`. (See the next 5-6 lines.)